### ISSUE TITLE
Node red node not populating with devices

### ISSUE THREAD

**Issue by hepcat72** [arupex/etekcity-smartplug#6]
I am not sure if you're responsible for node-red-contrib-etekcity-smartplug, but I just tried it out and I can't seem to set it up.  I entered my account info in the node, but the devices select list is not populated.  I've clicked the refresh button numerous times.  I've re-entered the info multiple times.  I tried both my email and "nickname" in the username field.

I entered the same account info into the homebridge config.json and it worked fine (homebridge-vesync-v2).  And it's the same login info I entered in the vesync app.  It works in both those places, so I know I have the right account info.

Could it be special characters in the password that aren't supported by node-red-contrib-etekcity-smartplug or etekcity-smartplug?  Or do I need to restart node red for it to populate?

**Comment by dirwin517** [comment]
Sorry, I do not maintain the node-red code, but it looks like it does depend on this so an upgrade might fix it!

#### The issue is likely:
homebridge-vesync-v2 uses apparently a v2 of the API while this code is hitting the V1
unfortunately the homebridge implementation removes energy data so I'll likely need to do some real work to get this updated to v2 (vesyncs payloads can be nondescript and use nonsense units)

#### TLDR: This library uses v1 for now
* V1 was plaintext form data
* v2 passwords are md5'd
* I'll see what I can do to upgrade it
* if you dont need energy data you could probably mash the code from homebridge-vesync-v2 into the node-red app and get it to work

**Comment by hepcat72** [comment]
OK.  No rush then.  I was able to accomplish my goal by using a node from node-red-contrib-homebridge-automation, which basically sends commands from node red, through homebridge, to the vesync outlet.

Incidentally, did you find a repo for node-red-contrib-etekcity-smartplug where I could submit an issue?  I couldn't find one.

**Comment by bencollinz** [comment]
I need this to work as well. I have no clue what "mash the code from homebridge-vesync-v2 into the node-red app and get it to work" means.

Can you please help?

edit: I don't care about energy management
